.circleci/config.yml

```diff
@@ -397,6 +398,7 @@ commands:
               --device model=hammerhead,version=23,locale=fr,orientation=portrait \
               --device model=oriole,version=31,locale=de,orientation=landscape \
               --device model=redfin,version=30,locale=en,orientation=portrait \
+              --device model=redfin,version=33,locale=it,orientation=landscape \
```

Comment:
Maybe we should replace one of existing devise with the latest one? What do you thing about having 23, 30, and 33 ?

Reply:
Maybe 23 and 33 will be enough? Do we expect some platform specific issues between min and max supported? @mapbox/navigation-android , what do you think?

Reply:
I'd vote for 23, 31 and 33 (old API, Android 13 and Android 12 (the prev one)).